deploy/basic/README_MSZ.md

```diff
@@ -21,10 +21,37 @@ If using local Terraform or on CloudShell:
 - Run `terraform init` to init the terraform providers
 - Run `terraform apply` to create the resources on OCI
 
-You can also include Web Application Firewall (WAF), DNS or other components as desired:
+You can also include Web Application Firewall (WAF), DNS or other components as desired.  Note that a public certificate will be required for WAF, and thus requires a DNS zone be created.  See below for more specific instructions.
```

Comment:
WAF will require a domain name, which is a tall order for most people who are playing around with OCI. This should be optional.

Reply:
Completely agree - I will make it more obvious in README.  And detail the requirements.